runs/pingcheck.sh

```diff
@@ -11,7 +12,10 @@ DebugLog(){
 
 #get Gateway for Connection
 gateway=$(ip route get 1 | awk '{print $3;exit}')
-
+if (( $DEBUG >= 1 )); then
+	load=$(uptime |cut -d ' ' -f 13-15)
```

Comment:
Bei meinem Pi mit Stretch enthalten die Felder 14-16 die Auslastung.

Reply:
Komisch, auf meiner Standard+ mit original Stretch passt 13-15 auch.
Kannst du mal eine Ausgabe von uptime ohne cut bei dir machen und reinstellen

Reply:
Standard+ Stretch
pi@openWB:/var/www/html/openWB/runs $ uptime
 09:57:12 up 22 days, 15:06,  3 users,  load average: 0,85, 0,94, 1,08
pi@openWB:/var/www/html/openWB/runs $ uname -a
Linux openWB 4.14.71-v7+ #1145 SMP Fri Sep 21 15:38:35 BST 2018 armv7l GNU/Linux
pi@openWB:/var/www/html/openWB/runs $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 9 (stretch)"
NAME="Raspbian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
pi@openWB:/var/www/html/openWB/runs $

Reply:
Buster
pi@raspi4:~ $ uptime
 09:59:44 up 8 days, 12:10,  1 user,  load average: 2,91, 2,81, 2,27
pi@raspi4:~ $ uname -a
Linux raspi4 5.10.17-v7l+ #1403 SMP Mon Feb 22 11:33:35 GMT 2021 armv7l GNU/Linux
pi@raspi4:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
pi@raspi4:~ $

Reply:
Aber ich könnte auch noch abwandeln in 
pi@openWB:/var/www/html/openWB/runs $ uptime |cut -d ' ' -f 13-
0,52, 0,72, 0,94
Dann sind deine auch mit drin ;-).

Reply:
Aber ich habe auch gerade gesehen, dass das so nicht viel bringt. Wenn ich den Debug Level auf 1 setze, dann kommen bereits so viele Meldungen, dass nur ca. 15 Minuten im Logfile sind. Damit hilft das nicht weiter.

Reply:
Bei mir ist noch ein Leerzeichen am Zeilenanfang.
` 11:42:16 up 9 days,  3:12,  2 users,  load average: 3,30, 2,70, 1,81`

Reply:
echt komisch mit dem Leerzeichen. Habe eine bessere Lösung gefunden
cat /proc/loadavg |cut -d ' ' -f 1-3
Aber was meinst Du, soll ich die Überprüfung auf Debug raus nehmen? Selbst bei Debug=1 wird so viel geloggt, dass bei mir innerhalb 15 Minunten das Log wieder gekürzt wird. Dadurch macht das keinen Sinn. Alternative wäre eine andere Logdatei einzuführen.
VG
Det

Reply:
Als Langzeitauswertung macht es so keinen Sinn. Eine Alternative habe ich gerade nicht parat.

Reply:
Ich gebe jetzt auch bei Debug=0 aus. Dannhat man evlt. die Möglichkeit zu sehen, ob vor einem "Hänger" ein hoher Load war.